statistics/selectivity_test.go

```diff
@@ -372,31 +382,51 @@ func getRange(start, end int64) []*ranger.Range {
 	return []*ranger.Range{ran}
 }
 
-func (s *testStatsSuite) TestOutOfRangeEQEstimation(c *C) {
+func (s *testStatsSuite) TestOutOfRangeEstimation(c *C) {
```

Comment:
Please add some cases about unsigned columns to test whether it would cause overflow.